src/profile.c

```diff
@@ -136,77 +136,51 @@ void Profile_Print(RedisModule_Reply *reply, AREQ *req, bool timedout) {
       IndexIterator *root = QITR_GetRootFilter(&req->qiter);
       // Coordinator does not have iterators
       if (root) {
-        RedisModule_ReplyKV_Array(reply, "Iterators profile");
-          PrintProfileConfig config = {.iteratorsConfig = &req->ast.config,
-                                       .printProfileClock = profile_verbose};
-          printIteratorProfile(reply, root, 0, 0, 2, req->reqflags & QEXEC_F_PROFILE_LIMITED, &config);
-        RedisModule_Reply_ArrayEnd(reply);
+        RedisModule_Reply_SimpleString(reply, "Iterators profile");
+        PrintProfileConfig config = {.iteratorsConfig = &req->ast.config,
+                                     .printProfileClock = profile_verbose};
+        printIteratorProfile(reply, root, 0, 0, 2, req->reqflags & QEXEC_F_PROFILE_LIMITED, &config);
```

Comment:
If this is another format change, please add it to the PR comment

Reply:
This is unifying the reply flow of `RESP2` and `RESP3`, I added the note at the end of the description